src/app/modules/main/profile_section/contacts/module.nim

```diff
@@ -40,8 +40,8 @@ proc createItemFromPublicKey(self: Module, publicKey: string): Item =
   let contact =  self.controller.getContact(publicKey)
   let (name, image) = self.controller.getContactNameAndImage(contact.id)
 
-  return initItem(contact.id, name, image, contact.isMutualContact(), contact.isBlocked(),
-  contact.isContactVerified(), contact.isContactUntrustworthy())
+  return initItem(contact.id, name, contact.localNickname, image, contact.isMutualContact(), contact.isBlocked(),
```

Comment:
@MishkaRogachev in this case I think that `name` and `contact.localNickname` will always be the same since we have some logic when we're dealing with the `userNameOrAlias` of `ContactsDto`, check [here](https://github.com/status-im/status-desktop/blob/69ef645d6b600c80e855ede5e414cf7823b090ff/src/app_service/service/contacts/dto/contacts.nim#L87-L95).

Reply:
We already have localNickname in ContactsDto, why don't just use it?

Reply:
@MishkaRogachev so far we were displaying the same "display name" everywhere across the app. So logic for display name is in `userNameOrAlias` proc of `ContactsDto`, I am not sure what are requirements of this ticket, but you should be careful with the changes and they affects the entire app. So if you contact model should expose `name` and `localName`, then `localName` is actually the `localNickname` prop form the `ContactsDto` and that's clear, but in case of name we need to define what is a name, does it refer to `name`, `alias` or `displayName` prop of `ContactsDto`, or it should have some logic beneath. For example I see this as one of solutions (update `userNameOrAlias`):

```
proc userNameOrAlias*(contact: ContactsDto, excludeLocalName = false): string =
  if(not excludeLocalName and contact.localNickname.len > 0):
    result = contact.localNickname
  elif(contact.name.len > 0 and contact.ensVerified):
    result = prettyEnsName(contact.name)
  elif contact.displayName.len > 0:
    result = contact.displayName
  else:
    result = contact.alias
```

and then your line 43 should be like:
```
return initItem(contact.id, contact.userNameOrAlias(true), contact.localNickname, image, contact.isMutualContact(), contact.isBlocked(), false, contact.isContactVerified(), contact.isContactUntrustworthy())
```
got it?

Reply:
Got it, thank you for explanation)